DiscreetORM.ts

```diff
@@ -104,12 +105,25 @@ export class StoredClass implements ObjectListener<any>{
         this.discreet_sql_io.writeSQL(escaped_command);
         this.discreet_sql_io.writeNewTable(table_name);
     }
-
+    addRow(obj: any) {
+                let table_name = obj.constructor.name;
+                let command = "INSERT INTO " + table_name + "\n" + "VALUES (";
```

Comment:
command construction should use sqlstring to create escaped

Reply:
+1